source/MRMesh/MRMeshDistance.cpp

```diff
@@ -263,6 +263,36 @@ MeshSignedDistanceResult findSignedDistance( const MeshPart & a, const MeshPart
     return (signedRes.signedDist > 0.0f) ? MeshSignedDistanceResult{res.a, res.b, 0.0f} : signedRes;
 }
 
+MRMESH_API float findMaxDistanceSq( const MeshPart& a, const MeshPart& b, const AffineXf3f* rigidB2A, float maxDistanceSq )
+{
+    MR_TIMER;
+
+    const auto& meshVerts = b.mesh.points;
+
+    std::vector<float> distances( meshVerts.size() );
+
+    const auto objXf = rigidB2A ? *rigidB2A : AffineXf3f();
+
+    //call refmesh-point distance finder for each point
+    BitSetParallelFor( a.mesh.topology.getValidVerts(), [&] ( VertId i )
+    {
+        const auto p = objXf( meshVerts[VertId( i )] );
+        if ( auto pp = a.mesh.projectPoint( p ) )
```

Comment:
this function is depricated, pls use findProjection (with maxDistanceSq  parameter )